mobile/assets/i18n/fr-FR.json

```diff
@@ -142,21 +142,21 @@
   "control_bottom_app_bar_archive": "Archive",
   "control_bottom_app_bar_create_new_album": "Créer un nouvel album",
   "control_bottom_app_bar_delete": "Supprimer",
-  "control_bottom_app_bar_edit_location": "Edit Location",
-  "control_bottom_app_bar_edit_time": "Edit Date & Time",
+  "control_bottom_app_bar_edit_location": "Modifier localisation",
+  "control_bottom_app_bar_edit_time": "Modifier date & heure",
```

Comment:
```suggestion
  "control_bottom_app_bar_edit_location": "Modifier la localisation",
  "control_bottom_app_bar_edit_time": "Modifier la date & l'heure",
```

Reply:
Maybe change `&` to `et`. It's not really common to see an ampersand in a french text.

Reply:
From https://fr.wikipedia.org/wiki/Esperluette, "Its use in French is essentially limited to commercial and advertising use, for social reasons."